annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -266,7 +272,7 @@ protected void validateTextualBody(Body body, boolean isLanguageMandatory) throw
     }
 
     protected void validateTagWithValue(Body body) throws ParamValidationException {
-
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.